MultiPage-Clone/index.html

```diff
@@ -4,18 +4,107 @@
+        rel="stylesheet">
     <title>Responsive grid project</title>
 </head>
-
 <body>
+<nav>
+    <div class="logo">
+        <div class="logo1">
+        <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQNgjV6Hxw34EU6zli1YKdNKY45CCCzDtqjKkUsX-uIh3nYTEMMFSL_CMb0Y00h16Hs0uo&usqp=CAU">
+        <span>Maya Nelson</span>
+        <span>/</span>
+        </div>
+        <p>PROJECT MANAGER</p>
+    </div>
+    <ul class="nav-links">
+        <li><a href="index.html">about me</a></li>
+        <li><a href="resume.html">resume</a></li>
+        <li><a href="#">project</a></li>
+        <li><a href="#">contact</a>
+    </ul>
+    <div class="navimage">
+        <div class="line1"></div>
+        <div class="line2"></div>
+        <div class="line3"></div>
+    </div>
+</nav>
 
-    <!-- Add your markup here -->
+<main>
+    <div class="left-side"></div>
+    <div class="right-side"></div>
+    <div class="center-box">
+    <div class="inner-left">
+    <img src="https://static.wixstatic.com/media/2e2a49_bd3a1599bc1b4265a01b2754b99b8536~mv2_d_5760_3840_s_4_2.jpeg/v1/crop/x_2703,y_43,w_2199,h_2205/fill/w_205,h_205,al_c,q_80,usm_0.66_1.00_0.01,enc_auto/AdobeStock_230407429.jpeg" alt="profile">
+    <div class="fist-heading">
+    <span>Maya</span>
+    <span>Nelson</span>
+    </div>
+    <p class="line"></p>
+    <div class="btn">
+        <button class="btn1">Resume</button>
+        <button class="btn2">Projects</button>
+    </div>
+    <p class="pp">PROJECT MANAGER</p>
+        <div class="social-icons">
+            <a href="https://www.facebook.com" target="_blank" class="social-link"><i class="fab fa-facebook-f"></i></a>
+            <a href="https://www.twitter.com" target="_blank" class="social-link"><i class="fab fa-twitter"></i></a>
+            <a href="https://www.linkedin.com" target="_blank" class="social-link"><i class="fab fa-linkedin-in"></i></a>
+            <a href="https://www.instagram.com" target="_blank" class="social-link"><i class="fab fa-instagram"></i></a>
+        </div>
+    </div>
+    <div class="inner-right">
+    <div>
+       <h1>Hello</h1>
+       <span>Here's who I am & what I do</span>
+    </div>
+       <div class="btn">
+       <button class="btn1">Resume</button>
+       <button class="btn2">Projects</button>
+       </div>
+       <div class="para">
+       <p>I'm a paragraph. Click here to add your own text and edit me. It’s easy. Just click “Edit Text” or double click me to
+    add your own content and make changes to the font.</p>
+    <p>I’m a great place for you to tell a story and let your users know a little more about you.</p>
+    </div>
+    </div>
+    </div>
+</main>
+<footer>
+    <div class="fot-left">
+        <span>© 2035 by Maya Nelson.</span>
+        <span>Powered and secured by <u>Wix</u></span>  
+    </div>
+    <div class="fot-right">
+        <div class="one">
+            <p>Call</p>
+            <span>123-456-7890</span>
+        </div>
+        <div class="second">
```

Comment:
What was the difficulty with using a single class instead of 3 separate classes: one, second, third?
You need to be able to add similar html elements without changing css.

Reply:
Same story for footer-left and footer-right.
Have you tried to use a single class for that?

Reply:
Class means "one of". For example "one of footer items".
If it doesn't read like this, there is something wrong probably.
"One of footer right" doesn't read really well. There is just one footer right component.

Reply:
I'd recommend to check for all cases when classes are used this way and to try to change it to us a single class. 
If one element should look differently, for example an active navigation link, it gets an extra class "active".